Gems/OpenXRVk/Code/Include/OpenXRVk/InputDeviceXRController.h

```diff
@@ -8,6 +8,8 @@
 
 #pragma once
 
+#include <AzCore/Console/IConsole.h>
```

Comment:
I think you can move `#include <AzCore/Console/IConsole.h>` to the cpp now.

Reply:
Done.